mem-21-a.xml

```diff
@@ -7,7 +7,7 @@
       <column name="definition_de">der Vokal {a:sen:nolink}</column>
       <column name="definition_fa"></column>
       <column name="definition_sv">klingonskans {a:sen:nolink}-ljud</column>
-      <column name="definition_ru">буква {a}</column>
+      <column name="definition_ru">буква {a:sen:nolink}</column>
```

Comment:
Is it unusual to distinguish consonants and vowels in Russian? Just wondering why you didn't keep that distinction, which is in the English definitions.